shortcircuit-api/src/main/java/com/powsybl/shortcircuit/ShortCircuitBusResults.java

```diff
@@ -6,28 +6,31 @@
  */
 package com.powsybl.shortcircuit;
 
-import java.util.Objects;
-
 /**
  * @author Coline Piloquet <coline.piloquet at rte-france.com>
  */
 public class ShortCircuitBusResults {
-    // FIXME
-    // delta voltages are missing for the moment,
-    // but not useful for the global design.
+
+    // VERSION = 1.0 voltageLevelId, busId, voltage
+    // VERSION = 1.1 voltageDropProportional
+    public static final String VERSION = "1.1";
 
     private final String voltageLevelId;
 
     private final String busId;
 
     private final FortescueValue voltage;
 
+    private final double voltageDropProportional;
+
     public ShortCircuitBusResults(String voltageLevelId,
                                   String busId,
-                                  FortescueValue voltage) {
+                                  FortescueValue voltage,
+                                  double voltageDropProportional) {
         this.voltageLevelId = voltageLevelId;
         this.busId = busId;
-        this.voltage = Objects.requireNonNull(voltage);
+        this.voltage = voltage;
```

Comment:
Does it make sense to have withVoltageDropProfileResult and withVoltageProfileResult set to different booleans?